var/spack/repos/builtin/packages/delphes/package.py

```diff
@@ -38,25 +39,18 @@ class Delphes(CMakePackage):
     version('3.0.6', sha256='9e225731d57d2a76d35886841f8eff121bb3a45560b16077bd8c351151581d88')
     version('3.0.5', sha256='ab64ec6d2476fbfa40562e7edb510a8ab4c4fe5be77a4353ebf315c2af181a80')
 
-    depends_on('cmake', type='build')
-    depends_on('root cxxstd=14', when='cxxstd=14')
-    depends_on('root cxxstd=17', when='cxxstd=17')
+    variant('pythia8', default=True,
+            description="build with pythia8")
 
-    variant('build_type', default='Release',
```

Comment:
Does this support all default CMake build types now (Release, Debug, RelWithDebInfo, MinSizeRel)? The default for CMake packages in Spack is `RelWithDebInfo`.

Reply:
I think when I wrote this recipe I wasn't aware that this variant is added by default to `CMakePackage` - there was never any need to restrict it :)